lib/http-conditional.php

```diff
@@ -80,7 +80,7 @@
  * @return bool True if the connection can be closed (e.g.: the client has already the latest version), false if the new content has to be send to the client.
  */
 function httpConditional(int $UnixTimeStamp, int $cacheSeconds = 0, int $cachePrivacy = 0, bool $feedMode = false, bool $compression = false, bool $session = false): bool {
-	if (headers_sent()) return false;
+	if (headers_sent() || (defined('NO_CACHE') && constant('NO_CACHE') === true)) return false;
```

Comment:
I would rather like to avoid modifying this file if possible, which is meant to be a standalone one.
I would prefer keeping the `if (!file_exists(DATA_PATH . '/no-cache.txt'))` test (also to make it easier to understand, in my opinion).